client/web/src/enterprise/batches/detail/BatchChangeStatsCard.story.tsx

```diff
@@ -39,22 +39,29 @@ export const Draft: Story = () => (
     </WebStory>
 )
 
-export const Open: Story = () => (
+export const Open: Story<OpenArgs> = args => (
     <WebStory>
         {props => (
             <BatchChangeStatsCard
                 {...props}
                 batchChange={{
                     changesetsStats: {
                         __typename: 'ChangesetsStats',
-                        closed: 10,
-                        deleted: 10,
-                        merged: 10,
-                        draft: 5,
-                        open: 10,
-                        total: 100,
-                        archived: 18,
-                        unpublished: 55,
+                        closed: args.closed,
+                        deleted: args.deleted,
+                        merged: args.merged,
+                        draft: args.draft,
+                        open: args.open,
+                        total:
+                            args.closed +
+                            args.deleted +
+                            args.merged +
+                            args.draft +
+                            args.open +
+                            args.archived +
```

Comment:
```suggestion
                            args.merged +
                            args.draft +
                            args.open +
```

Should `archived` & `deleted` changesets still be part of the total here? I think it shouldn't.

Reply:
I believe they should be here, actually. This is supposed to mirror the data we'd get from the backend, where `total` is literally just `SELECT COUNT(*) FROM CHANGESETS`.